lib/dom.js

```diff
@@ -53,6 +53,40 @@ declare class FileList {
     [index: number]: File;
 }
 
+/* DataTransfer */
+
+declare class DataTransfer {
+  dropEffect: string;
+  effectAllowed: string;
+
+  items: DataTransferItemList; // readonly
+
+  setDragImage(image: Element, x: number, y: number): void;
+
+  /* old interface */
+  types: Array<string>; // readonly
+  getData(format: string): string;
+  setData(format: string, data: string): void;
+  clearData(format?: string): void;
+  files: FileList; // readonly
+}
+
+declare class DataTransferItemList {
+  length: number; // readonly
+  [index: number]: DataTransferItem;
+  add(data: string, type: string): ?DataTransferItem;
+  add(data: File): ?DataTransferItem;
+  remove(index: number): void;
+  clear();
```

Comment:
here's another that's missing a return type. you can run the tests to double check, `./runtests.sh bin/flow dom`